src/main/java/plugily/projects/buildbattle/arena/ArenaManager.java

```diff
@@ -172,7 +172,7 @@ public static void joinAttempt(Player player, BaseArena arena) {
 
     player.setExp(1);
     player.setFoodLevel(20);
-    player.setHealth(20.0);
+    player.setHealth(player.getAttribute(Attribute.GENERIC_MAX_HEALTH).getBaseValue());
```

Comment:
`LivingEntity#getAttribute` is nullable. Sometime ago we got 5 error reports with same situations.
```suggestion
    MiscUtils.getEntityAttribute(player, Attribute.GENERIC_MAX_HEALTH).ifPresent(ai -> {
        player.setHealth(ai.getBaseValue()));
    });
```

Reply:
Ah ok